src/KafkaFlow.Retry/Durable/Definitions/Builders/Polling/RetryDurablePollingDefinitionBuilder.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace KafkaFlow.Retry
+{
+    using KafkaFlow.Retry.Durable.Definitions.Polling;
+
+    public class RetryDurablePollingDefinitionBuilder : PollingDefinitionBuilder<RetryDurablePollingDefinitionBuilder>
+    {
+        protected int expirationIntervalFactor = 1;
+        protected int fetchSize = 256;
+
+        public RetryDurablePollingDefinitionBuilder WithExpirationIntervalFactor(int expirationIntervalFactor)
+        {
+            this.expirationIntervalFactor = expirationIntervalFactor;
+            return this;
+        }
+
+        public RetryDurablePollingDefinitionBuilder WithFetchSize(int fetchSize)
```

Comment:
Perhaps we should set the admissible range of values for `fetchSize` and add a guard to validate it?

Reply:
We are already using a guard for positive values for this in the `RetryDurablePollingDefinition` constructor. I think we should limit the maximum value, it's the developer call when installing the retry extension IMO.